doc/classes/EditorNode3DGizmo.xml

```diff
@@ -12,8 +12,9 @@
 		<method name="_commit_handle" qualifiers="virtual">
 			<return type="void" />
 			<argument index="0" name="id" type="int" />
-			<argument index="1" name="restore" type="Variant" />
-			<argument index="2" name="cancel" type="bool" />
+			<argument index="1" name="secondary" type="bool" />
```

Comment:
Some docs should be added to define what `secondary` means in these methods. Not sure if it should be repeated in all methods but it should at least be described once so that users know what to make of it.